operators/couchdb-operator/2.0.1/metadata/annotations.yaml

```
@@ -0,0 +1,7 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: v2.1
+  operators.operatorframework.io.bundle.channels.v1: stable,v2.1  
```

if you want to publish it only on v2.1 then you cannot add stable here. 
```suggestion
   operators.operatorframework.io.bundle.channels.v1: v2.1  
```